app/graph/mutations/api_key_mutations.rb

```diff
@@ -0,0 +1,32 @@
+module ApiKeyMutations
+  MUTATION_TARGET = 'api_key'.freeze
+  PARENTS = ['team'].freeze
+
+  module CreateFields
+    extend ActiveSupport::Concern
+
+    included do
+      argument :title, GraphQL::Types::String, required: false
+      argument :description, GraphQL::Types::String, required: false
+    end
+  end
+
+  class Create < Mutations::CreateMutation
+    include CreateFields
+
+    def resolve(**inputs)
+      bot_user = BotUser.new
+      bot_user.name = inputs[:title]
+      bot_user.login = inputs[:title]
+      bot_user.save!
+
+      bot_user.api_key.title = inputs[:title]
+      bot_user.api_key.description = inputs[:description]
+      bot_user.api_key.save!
```

Comment:
Will be more clear to get `api_key` then do a save for api_key object.
i.e.
api_key = bot_user.api_key
api_key.title = inputs[:title]
api_key.description = inputs[:description]
 api_key.save!